packages/builder/src/builderStore/store/frontend.js

```diff
@@ -601,6 +601,28 @@ export const getFrontendStore = () => {
         // Finally try an external table
         return validTables.find(table => table.sourceType === DB_TYPE_EXTERNAL)
       },
+      processNestedSettings: enrichedComponent => {
+        const componentPrefix = "@budibase/standard-components"
+
+        if (enrichedComponent?._component == `${componentPrefix}/formblock`) {
+          // Use default config if the 'buttons' prop has never been initialised
+          if (!("buttons" in enrichedComponent)) {
+            enrichedComponent["buttons"] =
+              Utils.buildDynamicButtonConfig(enrichedComponent)
+            //Ensure existing Formblocks position their buttons at the top.
```

Comment:
Very very very NAB:  Lack of a space after a comment here compared with all the other comments you added.